### PR TITLE
Setup project

### PR DESCRIPTION
**Pull request by KanzaTahreem**
In this pull request, I 
- Set up the environment and tools needed to develop a React application.
- Set up and install linters.
- The application is running in the browser without errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
...
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+
+### Deployment
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+
+### `npm run build` fails to minify
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
```
Comment (zieeco) [inline README.md]: - Let's improve our readme to be more professional. you can use the [microverse readme template](https://github.com/microverseinc/readme-template) to update it by adding the missing headers like AUTHOR, GETTING STARTED, etc.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @KanzaTahreem,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- Project setup correctly :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,86 @@
...
+- Webpack
+- React
+- [Linters](https://github.com/microverseinc/linters-config/tree/master/react-redux)
+
+## Live Demo Link
+
+[Not Available!]()
+
+## Getting Started
+
```
Comment (fobadara) [inline README.md:24]: [OPTIONAL] You should include instructions like `git clone ...`. You can check out [this repo](https://github.com/fobadara/math-magicians)

**Review by fobadara — APPROVED** [review]
Hi @KanzaTahreem, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (KanzaTahreem) [reply README.md:24]: Noted!😃

**Pull request merged**